tests/utils/setup.rs

```diff
@@ -103,6 +105,9 @@ impl<'a> Setup<'a>
 
 impl<'a> Drop for Setup<'a> {
     fn drop(&mut self) {
+        self.pool_handle.map(|handle| {
```

Comment:
I think an if/let statement would be more appropriate. We don't need to actually map the value.